source/reference/ejson/deserialize.txt

```diff
@@ -0,0 +1,200 @@
+.. _mongosh-ejson-deserialize:
+
+===================
+EJSON.deserialize()
+===================
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. facet::
+   :name: programming_language
+   :values: shell
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. include:: /includes/links-urls/ejson.rst
+
+The ``EJSON.deserialize()`` method converts Extended JSON objects to
+JavaScript objects.
+
+Syntax
+------
+
+The method has this syntax:
+
+.. code-block:: javascript
+   :copyable: false
+
+   EJSON.deserialize( object, [ options ] )
+
+Method Fields
+-------------
+
+The method takes the following fields:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Necessity
+     - Description
+
+   * - ``object``
+     -  EJSON object
+     -  Required
+     -  EJSON object to convert. For example, an array of documents.
+
+   * - ``options``
+     -  string
+     -  Optional
+     -  Modifies the output object :ref:`types <mongo-shell-data-type>`.
+        The only option is ``{ relaxed: <boolean> }``.
+ 
+        .. list-table::
+           :header-rows: 1
+           :widths: 30 70
+
+           * - Boolean Value
+             - Description
+
+           * - ``true``
+             - Return JSON object types. Default is ``true``.
```

Comment:
```suggestion
             - Return plain JavaScript object types. Default is ``true``.
```

The return value isn’t JSON, so I’m not sure if “JSON object types” really makes the most sense here.